recipes/skillbridge/skillbridge.init.il

```diff
@@ -0,0 +1,13 @@
+; Initializes SkillBridge in Virtuoso when installed in a Viper environment using Virtue
+; http://www.cascode-labs.org/viper/
+; http://www.cascode-labs.org/virtue/
+
+printf("Initializing skillbridge\n")
+
+let((file_path dir_path)
+	file_path = simplifyFilename(get_filename(piport))
+	dir_path = strcat("/" buildString(reverse(cdr(reverse(parseString(file_path "/")))) "/"))
+	loadi(strcat( dir_path "/python_server.il"))
+)
+
+printf("  Done, Initialized skillbridge\n")
```

Comment:
Can you explain a bit why this is needed? I'm not familiar with the setup.

Reply:
This is a SKILL script which initializes the server in the Virtuoso design environment.  If you start Virtuoso from a  Conda environment setup then it will initialize the Python to SKILL server.

Reply:
@bastianzim Do you have any further questions or can it be merged?

Reply:
Thanks for the explanation. I would like to get a second opinion here as I'm not familiar enough with this to review it.

@ocefpaf Would you mind having a look at this?

Reply:
I'm not sure I understand why this and the build.sh are needed. What happens in a simple pip install of this pakcage?

Reply:
I have to confess that b/c this is a python library I'd rather you shipped those extra files in the lib itself rather than copying than like this. There is a disconnect between those who install it from PyPI and conda. Or, if those scripts only makes sense for those using Cadence Virtuoso, they should be shipped as a separate package. With that said I don't have a skin in the game b/c I'm not a user of any of those libraries. Let's merge and hopefully we can improve this situation in the future to ship a better experience to users.